src/wallet/scriptpubkeyman.h

```diff
@@ -555,6 +560,9 @@ class DescriptorScriptPubKeyMan : public ScriptPubKeyMan
     //! keeps track of whether Unlock has run a thorough check before
     bool m_decryption_thoroughly_checked = false;
 
+    //! Number of pre-generated keys/scripts (part of the look-ahead process, used to detect payments)
+    uint64_t m_keypool_size GUARDED_BY(cs_desc_man){DEFAULT_KEYPOOL_SIZE};
```

Comment:
Sorry for not being explicit in my previous comments. There are more cases where `uint64_t` could be replaced with `int64_t`:
```suggestion
    int64_t m_keypool_size GUARDED_BY(cs_desc_man){DEFAULT_KEYPOOL_SIZE};
```

and parameter `keypool_size` in (member) functions.

Reply:
np, small miscommunication. Done now, all of them tackled now. Thanks!